### PR TITLE
Fixed: #4092 Names are not in the standard BibTex format.

### PR DESCRIPTION
**Pull request by 1160300509**
Fixed: #4092

----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

**Comment by tobiasdiez** [comment]
@stefan-kolb @Siedlerchr Do you really think we should be forgiving with respect to the space? I'm fine with both but we should think a second about it...

Code-wise tests are missing, otherwise it looks good.

File: src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java [hunk 1]
```diff
@@ -33,13 +33,16 @@ public PersonNamesChecker(BibDatabaseContext databaseContext) {
 
         // Remove all brackets to handle corporate names correctly, e.g., {JabRef}
         value = new RemoveBrackets().format(value);
+
+        // put ' ' after every name, e.g.,L.C. Brand -> L. C. Brand
+        String checkFormat = value.replaceAll("\\.", ". ").replaceAll("\\  ", " ");
```
Comment (stefan-kolb) [inline src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java]: Why do we need `replaceAll("\\ ", " ")`

File: src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java [hunk 2]
```diff
@@ -33,13 +33,16 @@ public PersonNamesChecker(BibDatabaseContext databaseContext) {
 
         // Remove all brackets to handle corporate names correctly, e.g., {JabRef}
         value = new RemoveBrackets().format(value);
+
+        // put ' ' after every name, e.g.,L.C. Brand -> L. C. Brand
```
Comment (stefan-kolb) [inline src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java]: put space after...

**Comment by stefan-kolb** [comment]
@tobiasdiez what do you mean?

File: src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java (hunk 1 above)
Comment (1160300509) [reply src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java]: If we don't need `replaceAll("\\ ", " ")` , for example, `L.C. Brand` ,`checkFormat` will be   `L. C. Briand` ( there are two spaces between `L.` and `C.` , `C.` and `Brand`, not one space). So there's also a warning message. We use `checkFormat` to help check the name format.

**Comment by tobiasdiez** [comment]
@stefan-kolb I'm not sure if we really should change the behavior of the check. As far as I'm aware it is indeed the usual convention to add a space after the initial (e.g. `L. C. Brand` instead of `L.C. Brand`).

**Comment by stefan-kolb** [comment]
Hm yeah so this should not go into the checker but to the normalize names functionality?!

**Comment by tobiasdiez** [comment]
I thnik we should just leave the check as it was.

**Pull request closed without merging**